### PR TITLE
Repairing data, tranforming data, create README.md.

### PR DESCRIPTION
**Pull request by K4liber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# j-pet-multiphoton-classification
+Data transformation and classificiation methods for J-PET tomography.
+
+## Repairing data
+**Requirements**: python3.6 and the required packages  
+**Input format**: CSV  
+**Input columns**: Discribed in the function *dataFrameNames()* at */RepairingData/repairingData.py* script
```
Comment (wkrzemien) [inline README.md]: Discribed - >Described

File: RepairingData/repairingData.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+data['dt'] = data.apply (lambda row: row['t1'] - row['t2'], axis=1)
+data[['RX1','RY1','RZ1']] = data.apply(lambda row: pd.Series(emissionPoint(row)), axis=1)
+data['emissionDistance'] = data.apply(lambda row:distance(row), axis=1)
+data['class'] = data.apply(lambda row:reClass(row), axis = 1)
+data = data.apply(lambda row:shuffleTheOrder(row), axis = 1)
+data = data.drop(['dt', 'RX1', 'RY1','RZ1', 'emissionDistance'], axis = 1)
+data.to_csv(
+    pathToDataSave + fileName + 'REPAIRED_' + "part" + part, 
+    header=False, index=False, sep='\t'
+)
```
Comment (wkrzemien) [inline RepairingData/repairingData.py]: endline missing

File: TranformingData/transformingData.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        rX1,    # Reconstruction point - X cord
+        rY1,    # Reconstruction point - Y cord
+        rZ1,    # Reconstruction point - Z cord
+        rError, # Difference beetwen source point and recontructed point
+        volD,   # Volumes indexes difference
+        lorL,   # LOR length
+        deg3D,  # Angle beetwen lines (in XYZ geometry) connecting detection points with the center of detector
+        deg2D,  # Angle beetwen lines (in XY geometry) connecting detection points with the center of detector
+        rL,     # Distance beetween reconstructed point and the center of detector
+        eSum    # Sum of the detecions energies
```
Comment (wkrzemien) [inline TranformingData/transformingData.py]: detection

File: TranformingData/transformingData.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    )
+
+data = pd.read_csv(pathToData + fileName + "REPAIRED_part" + part, sep = "\t", names=dataFrameNames())
+data[['dt','rX1','rY1','rZ1','rError','volD','lorL','deg3D','deg2D','rL','eSum']] = data.apply(lambda row: pd.Series(featureEngineering(row)), axis=1)
+pickle.dump(data, open(pathToData + fileName + 'PREPARED_part' + part, 'wb'))
```
Comment (wkrzemien) [inline TranformingData/transformingData.py]: endline missing

File: RepairingData/repairingData.py [hunk 5]
```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3.6
+
+import pandas as pd
+import numpy as np
+import random as rnd
+import sys
+import math
+import random
```
Comment (wkrzemien) [inline RepairingData/repairingData.py:7]: you have already random above as rnd

File: RepairingData/repairingData.py (hunk 2 above)
Comment (wkrzemien) [inline RepairingData/repairingData.py]: endline missing

**Review by wkrzemien — COMMENTED** [review]
Nice. just small comments

File: RepairingData/repairingData.py (hunk 2 above)
Comment (K4liber) [reply RepairingData/repairingData.py]: Corrected.

File: RepairingData/repairingData.py (hunk 5 above)
Comment (K4liber) [reply RepairingData/repairingData.py:7]: Corrected.

File: TranformingData/transformingData.py (hunk 4 above)
Comment (K4liber) [reply TranformingData/transformingData.py]: Corrected.

File: TranformingData/transformingData.py (hunk 3 above)
Comment (K4liber) [reply TranformingData/transformingData.py]: Corrected.

File: RepairingData/repairingData.py (hunk 2 above)
Comment (K4liber) [reply RepairingData/repairingData.py]: Corrected.

File: README.md (hunk 1 above)
Comment (K4liber) [reply README.md]: Corrected.

**Pull request merged**